cdxev/__main__.py

```diff
@@ -709,6 +711,49 @@ def create_build_public_bom_parser(
     return parser
 
 
+# noinspection PyUnresolvedReferences,PyProtectedMember
+def create_init_sbom_parser(
+    subparsers: argparse._SubParsersAction,
+) -> argparse.ArgumentParser:
+    parser = subparsers.add_parser(
+        "init-sbom",
+        help=("Provides the first draft of an SBOM for manual completion."),
+    )
+    submitted_values = parser.add_argument_group(
+        "target",
+        description=(
+            "Submitted values that will be written into the SBOM draft."
+            "Field values like the name and version of the software (--name and --version), "
+            "the supplier of the software (--supplier-software) "
+            "or the supplier off the SBOM (--supplier-sbom) "
+            "can be submitted to the program and will be written into the provided draft."
+        ),
+    )
+    submitted_values.add_argument(
+        "--name",
+        metavar="<name>",
+        help=("Name of the software the SBOM belongs to."),
```

Comment:
I would prefer to use the description of either the [CycloneDX Specification](https://cyclonedx.org/docs/1.6/json/#metadata_component_name) or from the [NTIA minimum elements](https://www.ntia.gov/files/ntia/publications/sbom_minimum_elements_report.pdf). Same applies for the other `help`-description

Reply:
Changed it from 
"The name of the component." to "The name of the metadata component."
Since cdx does not differentiate and the description should be unambiguously for the users sake.

Reply:
@CBeck-96 agree, better adjust it to "The name of the component described by the SBOM".